plugins/main/loopback/loopback.go

```diff
@@ -78,6 +81,18 @@ func main() {
 }
 
 func cmdCheck(args *skel.CmdArgs) error {
-	// TODO: implement
-	return nil
+	args.IfName = "lo"
```

Comment:
There's a comment on the other places we force this value: probably good to have the same here.

Reply:
thanks, will update it